Makefile.org

```diff
@@ -685,7 +685,7 @@ install_docs:
 		(cd `$(PERL) util/dirname.pl $$i`; \
 		pod2man \
 			--section=$$sec --center=OpenSSL \
-			--release=$(VERSION) `basename $$i`) \
+			--release="OpenSSL $(VERSION)" `basename $$i`) \
```

Comment:
Ah. I'm doing this all on OS X, where most of the system commands are BSD. Style seems different over here, favoring package + version or just package name.

Reverted.